R/columns.R

```diff
@@ -1,3 +1,78 @@
+#' Set column definitions
+#'
+#' @param widget A [tabulator()] HTML widget.
+#' @param columns The names of the columns the formatter is applied to.
+#' @param title A title that will be displayed for the columns.
+#' @param formatter A formatter definition as generated by a `formatter_*()` function
+#' @param hoz_align (character): The horizontal alignment of the column.
+#' @param ... You can add arguments that will be passed on as updates to the column definition.
+#' Checkout the documentation at \url{https://tabulator.info/docs/}
+#' @returns The updated [tabulator()] HTML widget
+#' @example examples/formatters/formatter_html.R
+#' @export
+set_columns <- function(widget,
```

Comment:
I prefer singular names: `set_column(column, ...)` because R is generally vectorized, but it`s ok for me.